trillian/ctfe/ct_server/main.go

```diff
@@ -140,6 +142,23 @@ func main() {
 		http.Handle("/metrics", promhttp.Handler())
 	}
 
+	if *getSTHInterval > 0 {
+		// Regularly update the internal STH for each log so our metrics stay up-to-date with any tree head
+		// changes that are not triggered by us.
+		for _, c := range cfg {
```

Comment:
I'm wondering whether this should be a single background operation that cycles round all the logs. I don't have a firm feeling that it's overall better though.